<?xml version="1.0"?>
<launch>    
    <node name="dodo_detector_ros" pkg="dodo_detector_ros" type="detector.py" output="screen">
        <rosparam command="load" file="$(find dodo_detector_ros)/config/main_config.yaml"/>
        <param name="image_topic" value="/camera/rgb/image_raw" />
        <param name="point_cloud_topic" value="/camera/depth_registered/points" />
    </node>
</launch>
